modules/apps/portal-settings/portal-settings-web/src/main/java/com/liferay/portal/settings/web/internal/portlet/action/EditCompanyMVCActionCommand.java

```diff
@@ -219,6 +221,20 @@ private void _updateCompany(ActionRequest actionRequest) throws Exception {
 		UnicodeProperties unicodeProperties = PropertiesParamUtil.getProperties(
 			actionRequest, "settings--");
 
+		URLValidator urlValidator = _urlValidatorSnapshot.get();
```

Comment:
@LisztLiszt Please sort -- put the validation after the `PropsKeys.ADMIN_EMAIL_FROM_ADDRESS` check.